apps/examples/wifi_test/wifi_test.c

```diff
@@ -0,0 +1,309 @@
+ *    distribution.
+ * 3. Neither the name NuttX nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <sys/socket.h>
+
+#include <dm/dm_connectivity.h>
+#include <dm/dm_error.h>
+
+/****************************************************************************
+ * Definitions
+ ****************************************************************************/
+#define DEVICE_NAME 	"ATK/002"	// "ATK" or "STA"
+#define SERVER_IP	"192.168.1.117"
+#define SERVER_PORT 	5987
+
+#define WIFI_BSSID_SIZE	17
+#define BUF_MAX_SIZE 	128
+
+#ifdef __has_include
+#if __has_include("ap_check_list.txt")
+#define AP_CHECK_LIST
+char aps_list[] = {
+#include "ap_check_list.txt"
+};
+
+typedef struct {
+	int id;
+	char bssid[WIFI_BSSID_SIZE + 1];
+	int8_t rssi;
+} Ap_t;
+
+static Ap_t *aps = NULL;
+static int aps_cnt = 0;
+#else
+#undef AP_CHECK_LIST
+#endif
+#endif
+
+static int isWiFiConnected;
+/****************************************************************************
+ * Prototype Functions
+ ****************************************************************************/
+extern int wifi_test_main(int argc, char *argv[]);
+static int wifi_test_proc(int argc, char *argv[]);
+
+#ifdef AP_CHECK_LIST
+void parse_aps_list(void);
+#endif
+static void printScanResult(dm_scan_info_t *list);
+
+void WiFi_Link_Up(void);
+void WiFi_Link_Down(void);
+
+/****************************************************************************
+ * Private Data & Functions
+ ****************************************************************************/
+void WiFi_Link_Up()
+{
+	isWiFiConnected = 1;
+	printf("\n[WiFi] Connected!\n");
+}
+
+void WiFi_Link_Down()
+{
+	isWiFiConnected = 0;
+	printf("\n[WiFi] Disconnected!\n");
+}
+
+#ifdef AP_CHECK_LIST
+void parse_aps_list(void)
+{
+	char *ptr, *p;
```

Comment:
Don't define several variables at same line. This is a violation on Tizen Coding Style